# Assignment \#3 : Binary Systems

Kaimi Kahihikolo

Due: 11 February 2019

In [2]:
import numpy as np
import astropy.units as u
import astropy.constants as const

___
## (1) An Eclipsing Binary in the LMC

Data taken from Pietrzynski et al. 2009, ApJ, 697, 862. We will assume that the orbits are circular, which is OK, because the eccentricity is small. We also assume that the orbital inclination is 90 degrees, which again is close to the truth.

___
## (1.1) Orbital size, sum of stellar masses

The orbital period is 214.37 days. The semiamplitudes K are 32.65 and 33.67 km/s. **Calculate the orbital size (a1 + a2).** Recall, 
$$\begin{align} Pv_1 &= 2\pi a_1 \\  Pv_2 &= 2\pi a_2 \end{align}$$

In [53]:
P = 214.37 *u.day
K1 = 32.65 * u.km / u.s
K2 = 33.67 * u.km / u.s

In [75]:
a1 = ((P * K1)/(2*np.pi)).to('m')
a2 = ((P * K2)/(2*np.pi)).to('m')
a = a1 + a2

print(f"\t a1 + a2 = {a.to('AU').round(2)}")

	 a1 + a2 = 1.31 AU


Use this result to calculate the sum of the stellar masses. Recall,

$$P = \frac{2\pi}{\Omega}$$

where,

$$\Omega = \sqrt{\frac{G(M_1 + M_2)}{(a_1 + a_2)^3}}$$

Therefore, by solving for $(M_1 + M_2)$, we get,

$$M_1 + M_2 = \frac{4\pi^2 (a_1+a_2)^3}{G P^2}$$

In [55]:
M = (4.*np.pi**2*(a)**3)/(const.G*P**2).decompose()

print(f"\t M1 + M2 = {M}")

print(f"\t Which is about {(M/const.M_sun).round(2)} times more massive than the Sun.")
print("(I am just checking if the number looks reasonable)")

	 M1 + M2 = 1.2883682882482724e+31 kg
	 Which is about 6.48 times more massive than the Sun.
(I am just checking if the number looks reasonable)


___
## (1.2) Mass Ratio and Individual Masses

Calculate the mass ratio from the ratio of the semiamplitudes, and calculate the individual masses.

Recall,

$$\frac{v_2}{v_1} = \frac{M_1}{M_2}$$

In [56]:
mass_ratio = K2/K1

print(f"M1 / M2 = {mass_ratio}")

M1 / M2 = 1.0312404287901993


Recall, the individual masses can be computed by noticing:

$$\begin{align} M_1 &= \frac{M\cdot(a - a_1)}{a} \\ M_2 &= \frac{M\cdot(a - a_2)}{a}\end{align}$$

In [74]:
M1 = M*(a - a1)/a
print(f"\t M1 = {M1.to('M_sun').round(2)}")

M2 = M*(a - a2)/a
print(f"\t M2 = {M2.to('M_sun').round(2)}")

	 M1 = 3.29 solMass
	 M2 = 3.19 solMass


___
## (1.3) Angular Sizes and Distances to the LMC

The values of the apparent magnitude K (do not confuse with the semiamplitude K) are 14.895 and 15.446 for the two binary components. The values of the colors V-K are 1.843 and 1.749. The equation for the Barnes-Evans relation is

$$K + 5\log{\phi} = 2.76 + 0.252(V-K)$$

Calculate the angular sizes. The analysis of the light curve has provided the physical radii of the two stars, $26.06$ and $19.76$ solar radii. Calculate the distances to both stars.

## (2) Elliptical Orbits